R/get_census_api.R

```diff
@@ -5,8 +5,8 @@
 #' This function obtains U.S. Census data via the public API. User
 #' can specify the variables and region(s) for which to obtain data.
 #'
-#' @param data_url URL root of the API, including the question mark,
```

Comment:
Verify this change to docs please. @mdblocker

Reply:
<img width="1381" alt="image" src="https://github.com/kosukeimai/wru/assets/46643241/07ca5308-a15f-4eec-8d4c-31666d894f73">
Verified, sends me to the 2020 version of the API-endpoint (screen grab for confirmation)